res/res_pjsip_sdp_rtp.c

```diff
@@ -1900,6 +1944,9 @@ static int create_outgoing_sdp_stream(struct ast_sip_session *session, struct as
 		ast_format_cap_append_from_cap(caps, ast_stream_get_formats(stream), media_type);
 	}
 
+	/* Init the sample rates before we start adding them */
+	AST_VECTOR_INIT(&sampleRates, 0);
```

Comment:
I think an init of 1 makes most sense since 8kHz will be common, this can also fail